### ISSUE TITLE
Telitalálatos forduló

### ISSUE THREAD

**Issue by GuberPeter** [bertadenes/tippelde#15]
Ha valaki mind a 6 meccs végkimenetelét eltalálja a fordulóban (nem pontosan), kap 25 pontot.

Manuálisan szerkeszteném, de az összesítő táblázatba valahogy hozzá kellene tudnom adni (úgy, hogy látszódjon, miért járt).

**Comment by bertadenes** [comment]
a manualis szerkeztes adott, a megfelelo tournament-hez valo score-ba kell beirni. lehet ehhez irok egy oldat, hogy konnyebb legyen

**Comment by GuberPeter** [comment]
Scoreba beiras=x ponthoz hozzaadok 25ot? Mas nyoma nem lesz?

**Comment by bertadenes** [comment]
Nem, jelenleg nincs. Mennyire szeretnenk ezt izolaltan gyujteni? Lehet hogy minden fordulo-hoz egy tour-t kellene definialni? Hivhatjuk fordulonak is :D

**Comment by GuberPeter** [comment]
Szerintem az szerencsésebb lenne, igen. De akkor viszont fontos lenne egy kumulatív táblázat létrehozása is, előbb-utóbb.
